src/ckeditor.jsx

```diff
@@ -5,15 +5,36 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
+import Context from './context.jsx';
+import EditorWatchdog from '@ckeditor/ckeditor5-watchdog/src/editorwatchdog';
+import uid from './utils/uid';
 
 export default class CKEditor extends React.Component {
 	constructor( props ) {
 		super( props );
 
 		// After mounting the editor, the variable will contain a reference to the created editor.
 		// @see: https://ckeditor.com/docs/ckeditor5/latest/api/module_core_editor_editor-Editor.html
-		this.editor = null;
 		this.domContainer = React.createRef();
+		this.watchdog = null;
+
+		this._id = uid();
+	}
+
+	get editor() {
+		if ( this.watchdog ) {
+			return this.watchdog.editor;
+		}
+
+		if ( this.props.contextWatchdog ) {
+			try {
+				return this.props.contextWatchdog.getItem( this._id );
+			} catch ( err ) {
+				return null;
+			}
+		}
+
+		return null;
```

Comment:
The question is – how to get an editor instance when a user does not use Watchdog and Context features?

Reply:
Ok, I see that Watchdog is now built-in in the component.